<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<script src="https://2020.igem.org/Template:Waterloo/js/vendor?action=raw&ctype=text/javascript"></script>
	<link rel="stylesheet" href="https://2020.igem.org/Template:Waterloo/css/vendor?action=raw&ctype=text/css">
	<link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@400;600;700;800;900&display=swap" rel="stylesheet">
	<script>
		$(function() {
			$('header').load('https://2020.igem.org/Template:Waterloo/header?action=raw&ctype=text/javascript');
			$('mobileHeader').load('https://2020.igem.org/Template:Waterloo/headerMobile?action=raw&ctype=text/javascript');
			$('footer').load('https://2020.igem.org/Template:Waterloo/footer?action=raw&ctype=text/javascript');
			$('accessibility').load('https://2020.igem.org/Template:Waterloo/accessibility?action=raw&ctype=text/javascript');
		});
	</script>
	<link rel="stylesheet" type="text/css" href="https://2020.igem.org/wiki/index.php?title=Template:Waterloo/css/styles&action=raw&ctype=text/css">
	<script src="https://2020.igem.org/Template:Waterloo/js/wiki?action=raw&ctype=text/javascript"></script>
	<script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
</head>

<body>
	<div class="topElement"></div>
	<div id="igem-navbar-placeholder">
		<p>Placeholder for iGEM Navbar</p>
	</div>
	<header></header>
    <main id="top">
		<article>
			<mobileHeader></mobileHeader>
			<div class="pageHeader"> 
				<div class="headerCol">
					<h1 class="modelpage"> MODEL </h1>
				</div> 
				<div class="headerCol2">
					<img src='/images/buildings.svg'>
				</div>
				<div class="buildingsMobile">
					<img src='/images/buildings.svg'>
				</div>
			</div>

			<div class="pipecol">
				<svg width="227" height="3442" viewBox="0 0 227 3442" fill="none" overflow="visible" xmlns="http://www.w3.org/2000/svg">
					<rect class="proteinPipe" fill="#CDDCDD" fill-opacity="0" x="5" y="0" width="20" height="175"></rect>
					<rect class="processPipe" fill="#CDDCDD" fill-opacity="0" x="50" y="0" width="20" height="180"></rect>
					<path id="text-one" class="proteinPipe" d="M20 168.198H5.9V162.078C5.9 160.532 6.28667 159.338 7.06 158.498C7.83333 157.658 8.91333 157.238 10.3 157.238C11.6867 157.238 12.7733 157.658 13.56 158.498C14.3333 159.338 14.72 160.532 14.72 162.078V165.638H20L20 168.198ZM12.7 165.638V162.478C12.7 160.638 11.9 159.718 10.3 159.718C8.71333 159.718 7.92 160.638 7.92 162.478V165.638H12.7ZM20 155.151H5.9V149.031C5.9 147.485 6.27333 146.291 7.02 145.451C7.75333 144.611 8.78667 144.191 10.12 144.191C11.1733 144.191 12.0467 144.465 12.74 145.011C13.42 145.558 13.88 146.345 14.12 147.371C14.3333 146.691 14.8467 146.131 15.66 145.691L20 143.331V146.151L15.5 148.591C15.06 148.831 14.76 149.118 14.6 149.451C14.44 149.771 14.36 150.185 14.36 150.691V152.591H20V155.151ZM12.46 152.591V149.471C12.46 147.605 11.7 146.671 10.18 146.671C8.67333 146.671 7.92 147.605 7.92 149.471V152.591H12.46ZM20.2 135.218C20.2 136.578 19.9 137.771 19.3 138.798C18.7 139.811 17.86 140.598 16.78 141.158C15.6867 141.718 14.4067 141.998 12.94 141.998C11.46 141.998 10.18 141.718 9.1 141.158C8.02 140.598 7.18667 139.811 6.6 138.798C6 137.785 5.7 136.591 5.7 135.218C5.7 133.845 6 132.651 6.6 131.638C7.18667 130.625 8.02 129.838 9.1 129.278C10.18 128.718 11.4533 128.438 12.92 128.438C14.4 128.438 15.6867 128.718 16.78 129.278C17.86 129.838 18.7 130.625 19.3 131.638C19.9 132.651 20.2 133.845 20.2 135.218ZM18 135.218C18 133.925 17.5533 132.918 16.66 132.198C15.7667 131.478 14.5267 131.118 12.94 131.118C11.34 131.118 10.1 131.478 9.22 132.198C8.34 132.905 7.9 133.911 7.9 135.218C7.9 136.511 8.34 137.518 9.22 138.238C10.1 138.958 11.34 139.318 12.94 139.318C14.5267 139.318 15.7667 138.958 16.66 138.238C17.5533 137.518 18 136.511 18 135.218ZM20 123.294H8.04L8.04 128.134H5.9L5.9 115.894H8.04V120.734H20V123.294ZM20 114.272H5.9V104.892H7.94V111.812H11.82V105.332H13.86V111.812H17.96L17.96 104.892H20L20 114.272ZM20 102.339H5.9V99.7789H20V102.339ZM20 96.7139H5.9V94.7939L15.74 87.2139H5.9V84.8339H20V86.7339L10.12 94.3339H20V96.7139ZM20 76.3623H5.9L5.9 71.1223C5.9 68.7623 6.50667 66.9423 7.72 65.6623C8.93333 64.369 10.6733 63.7223 12.94 63.7223C15.1933 63.7223 16.9333 64.369 18.16 65.6623C19.3867 66.9423 20 68.7623 20 71.1223V76.3623ZM17.88 73.8023V71.2823C17.88 68.029 16.2333 66.4023 12.94 66.4023C9.66 66.4023 8.02 68.029 8.02 71.2823V73.8023H17.88ZM20 61.128H5.9L5.9 51.748H7.94V58.668H11.82V52.188H13.86V58.668H17.96L17.96 51.748H20L20 61.128ZM20.2 44.5297C20.2 45.6364 20.06 46.663 19.78 47.6097C19.4867 48.5564 19.0933 49.3564 18.6 50.0097L16.58 49.2297C17.0467 48.5764 17.4067 47.863 17.66 47.0897C17.9133 46.303 18.04 45.4497 18.04 44.5297C18.04 43.4764 17.8667 42.7097 17.52 42.2297C17.16 41.7497 16.7 41.5097 16.14 41.5097C15.6733 41.5097 15.3067 41.683 15.04 42.0297C14.7733 42.363 14.5533 42.943 14.38 43.7697L13.92 45.9697C13.3867 48.4764 12.1 49.7297 10.06 49.7297C9.18 49.7297 8.41333 49.4964 7.76 49.0297C7.10667 48.563 6.6 47.9164 6.24 47.0897C5.88 46.263 5.7 45.3097 5.7 44.2297C5.7 43.2697 5.84667 42.3697 6.14 41.5297C6.42 40.6897 6.82 39.9897 7.34 39.4297L9.26 40.2097C8.32667 41.3164 7.86 42.663 7.86 44.2497C7.86 45.1697 8.05333 45.8897 8.44 46.4097C8.82667 46.9297 9.33333 47.1897 9.96 47.1897C10.44 47.1897 10.8267 47.0297 11.12 46.7097C11.4 46.3897 11.62 45.843 11.78 45.0697L12.24 42.8897C12.52 41.5697 12.96 40.5897 13.56 39.9497C14.16 39.2964 14.96 38.9697 15.96 38.9697C16.8 38.9697 17.54 39.1964 18.18 39.6497C18.82 40.103 19.32 40.7497 19.68 41.5897C20.0267 42.4164 20.2 43.3964 20.2 44.5297ZM20 36.7725H5.9V34.2125H20V36.7725ZM20.2 24.3675C20.2 25.9275 19.9067 27.2475 19.32 28.3275C18.7333 29.3942 17.9 30.2075 16.82 30.7675C15.74 31.3275 14.46 31.6075 12.98 31.6075C11.4867 31.6075 10.2 31.3208 9.12 30.7475C8.02667 30.1742 7.18667 29.3475 6.6 28.2675C6 27.1875 5.7 25.8942 5.7 24.3875C5.7 23.4008 5.84667 22.4742 6.14 21.6075C6.42 20.7408 6.81333 20.0208 7.32 19.4475L9.32 20.3075C8.82667 20.9342 8.47333 21.5742 8.26 22.2275C8.04667 22.8808 7.94 23.6008 7.94 24.3875C7.94 25.8808 8.37333 27.0142 9.24 27.7875C10.1067 28.5475 11.3533 28.9275 12.98 28.9275C14.62 28.9275 15.8667 28.5342 16.72 27.7475C17.5733 26.9608 18 25.7942 18 24.2475C18 23.8075 17.9667 23.3608 17.9 22.9075C17.82 22.4408 17.7067 21.9808 17.56 21.5275H14.28V24.7675H12.38V19.3875H19.16C19.48 20.0408 19.7333 20.8142 19.92 21.7075C20.1067 22.6008 20.2 23.4875 20.2 24.3675ZM20 16.4209H5.9L5.9 14.5009L15.74 6.92094H5.9V4.54094H20V6.44094L10.12 14.0409H20V16.4209Z" fill="#003650"/>
					<path id="text-two" class="processPipe" d="M68 174.663H53.9V168.543C53.9 166.996 54.2867 165.803 55.06 164.963C55.8333 164.123 56.9133 163.703 58.3 163.703C59.6867 163.703 60.7733 164.123 61.56 164.963C62.3333 165.803 62.72 166.996 62.72 168.543V172.103H68V174.663ZM60.7 172.103V168.943C60.7 167.103 59.9 166.183 58.3 166.183C56.7133 166.183 55.92 167.103 55.92 168.943V172.103H60.7ZM68 161.616H53.9V155.496C53.9 153.95 54.2733 152.756 55.02 151.916C55.7533 151.076 56.7867 150.656 58.12 150.656C59.1733 150.656 60.0467 150.93 60.74 151.476C61.42 152.023 61.88 152.81 62.12 153.836C62.3333 153.156 62.8467 152.596 63.66 152.156L68 149.796V152.616L63.5 155.056C63.06 155.296 62.76 155.583 62.6 155.916C62.44 156.236 62.36 156.65 62.36 157.156V159.056H68V161.616ZM60.46 159.056V155.936C60.46 154.07 59.7 153.136 58.18 153.136C56.6733 153.136 55.92 154.07 55.92 155.936V159.056H60.46ZM68.2 141.683C68.2 143.043 67.9 144.236 67.3 145.263C66.7 146.276 65.86 147.063 64.78 147.623C63.6867 148.183 62.4067 148.463 60.94 148.463C59.46 148.463 58.18 148.183 57.1 147.623C56.02 147.063 55.1867 146.276 54.6 145.263C54 144.25 53.7 143.056 53.7 141.683C53.7 140.31 54 139.116 54.6 138.103C55.1867 137.09 56.02 136.303 57.1 135.743C58.18 135.183 59.4533 134.903 60.92 134.903C62.4 134.903 63.6867 135.183 64.78 135.743C65.86 136.303 66.7 137.09 67.3 138.103C67.9 139.116 68.2 140.31 68.2 141.683ZM66 141.683C66 140.39 65.5533 139.383 64.66 138.663C63.7667 137.943 62.5267 137.583 60.94 137.583C59.34 137.583 58.1 137.943 57.22 138.663C56.34 139.37 55.9 140.376 55.9 141.683C55.9 142.976 56.34 143.983 57.22 144.703C58.1 145.423 59.34 145.783 60.94 145.783C62.5267 145.783 63.7667 145.423 64.66 144.703C65.5533 143.983 66 142.976 66 141.683ZM68.2 125.74C68.2 127.207 67.9067 128.467 67.32 129.52C66.72 130.56 65.88 131.36 64.8 131.92C63.7067 132.48 62.42 132.76 60.94 132.76C59.46 132.76 58.18 132.48 57.1 131.92C56.02 131.36 55.1867 130.56 54.6 129.52C54 128.467 53.7 127.207 53.7 125.74C53.7 124.793 53.8467 123.9 54.14 123.06C54.4333 122.207 54.8533 121.507 55.4 120.96L57.44 121.8C56.9333 122.4 56.5667 123.013 56.34 123.64C56.1 124.267 55.98 124.94 55.98 125.66C55.98 127.087 56.4133 128.173 57.28 128.92C58.1333 129.667 59.3533 130.04 60.94 130.04C62.5267 130.04 63.7533 129.667 64.62 128.92C65.4867 128.173 65.92 127.087 65.92 125.66C65.92 124.94 65.8067 124.267 65.58 123.64C65.34 123.013 64.9667 122.4 64.46 121.8L66.5 120.96C67.0333 121.507 67.4533 122.207 67.76 123.06C68.0533 123.9 68.2 124.793 68.2 125.74ZM68 118.726H53.9V109.346H55.94V116.266H59.82V109.786H61.86V116.266H65.96V109.346H68V118.726ZM68.2 102.127C68.2 103.234 68.06 104.261 67.78 105.207C67.4867 106.154 67.0933 106.954 66.6 107.607L64.58 106.827C65.0467 106.174 65.4067 105.461 65.66 104.687C65.9133 103.901 66.04 103.047 66.04 102.127C66.04 101.074 65.8667 100.307 65.52 99.8273C65.16 99.3473 64.7 99.1073 64.14 99.1073C63.6733 99.1073 63.3067 99.2807 63.04 99.6273C62.7733 99.9607 62.5533 100.541 62.38 101.367L61.92 103.567C61.3867 106.074 60.1 107.327 58.06 107.327C57.18 107.327 56.4133 107.094 55.76 106.627C55.1067 106.161 54.6 105.514 54.24 104.687C53.88 103.861 53.7 102.907 53.7 101.827C53.7 100.867 53.8467 99.9673 54.14 99.1273C54.42 98.2873 54.82 97.5873 55.34 97.0273L57.26 97.8073C56.3267 98.914 55.86 100.261 55.86 101.847C55.86 102.767 56.0533 103.487 56.44 104.007C56.8267 104.527 57.3333 104.787 57.96 104.787C58.44 104.787 58.8267 104.627 59.12 104.307C59.4 103.987 59.62 103.441 59.78 102.667L60.24 100.487C60.52 99.1673 60.96 98.1873 61.56 97.5473C62.16 96.894 62.96 96.5673 63.96 96.5673C64.8 96.5673 65.54 96.794 66.18 97.2473C66.82 97.7007 67.32 98.3473 67.68 99.1873C68.0267 100.014 68.2 100.994 68.2 102.127ZM68.2 89.5102C68.2 90.6168 68.06 91.6435 67.78 92.5902C67.4867 93.5368 67.0933 94.3368 66.6 94.9902L64.58 94.2102C65.0467 93.5568 65.4067 92.8435 65.66 92.0702C65.9133 91.2835 66.04 90.4302 66.04 89.5102C66.04 88.4568 65.8667 87.6902 65.52 87.2102C65.16 86.7302 64.7 86.4902 64.14 86.4902C63.6733 86.4902 63.3067 86.6635 63.04 87.0102C62.7733 87.3435 62.5533 87.9235 62.38 88.7502L61.92 90.9502C61.3867 93.4568 60.1 94.7102 58.06 94.7102C57.18 94.7102 56.4133 94.4768 55.76 94.0102C55.1067 93.5435 54.6 92.8968 54.24 92.0702C53.88 91.2435 53.7 90.2902 53.7 89.2102C53.7 88.2502 53.8467 87.3502 54.14 86.5102C54.42 85.6702 54.82 84.9702 55.34 84.4102L57.26 85.1902C56.3267 86.2968 55.86 87.6435 55.86 89.2302C55.86 90.1502 56.0533 90.8702 56.44 91.3902C56.8267 91.9102 57.3333 92.1702 57.96 92.1702C58.44 92.1702 58.8267 92.0102 59.12 91.6902C59.4 91.3702 59.62 90.8235 59.78 90.0502L60.24 87.8702C60.52 86.5502 60.96 85.5702 61.56 84.9302C62.16 84.2768 62.96 83.9502 63.96 83.9502C64.8 83.9502 65.54 84.1768 66.18 84.6302C66.82 85.0835 67.32 85.7302 67.68 86.5702C68.0267 87.3968 68.2 88.3768 68.2 89.5102ZM68 76.3623H53.9V71.1223C53.9 68.7623 54.5067 66.9423 55.72 65.6623C56.9333 64.369 58.6733 63.7223 60.94 63.7223C63.1933 63.7223 64.9333 64.369 66.16 65.6623C67.3867 66.9423 68 68.7623 68 71.1223V76.3623ZM65.88 73.8023V71.2823C65.88 68.029 64.2333 66.4023 60.94 66.4023C57.66 66.4023 56.02 68.029 56.02 71.2823V73.8023H65.88ZM68 61.128H53.9V51.748H55.94V58.668H59.82V52.188H61.86V58.668H65.96V51.748H68L68 61.128ZM68.2 44.5297C68.2 45.6364 68.06 46.663 67.78 47.6097C67.4867 48.5564 67.0933 49.3564 66.6 50.0097L64.58 49.2297C65.0467 48.5764 65.4067 47.863 65.66 47.0897C65.9133 46.303 66.04 45.4497 66.04 44.5297C66.04 43.4764 65.8667 42.7097 65.52 42.2297C65.16 41.7497 64.7 41.5097 64.14 41.5097C63.6733 41.5097 63.3067 41.683 63.04 42.0297C62.7733 42.363 62.5533 42.943 62.38 43.7697L61.92 45.9697C61.3867 48.4764 60.1 49.7297 58.06 49.7297C57.18 49.7297 56.4133 49.4964 55.76 49.0297C55.1067 48.563 54.6 47.9164 54.24 47.0897C53.88 46.263 53.7 45.3097 53.7 44.2297C53.7 43.2697 53.8467 42.3697 54.14 41.5297C54.42 40.6897 54.82 39.9897 55.34 39.4297L57.26 40.2097C56.3267 41.3164 55.86 42.663 55.86 44.2497C55.86 45.1697 56.0533 45.8897 56.44 46.4097C56.8267 46.9297 57.3333 47.1897 57.96 47.1897C58.44 47.1897 58.8267 47.0297 59.12 46.7097C59.4 46.3897 59.62 45.843 59.78 45.0697L60.24 42.8897C60.52 41.5697 60.96 40.5897 61.56 39.9497C62.16 39.2964 62.96 38.9697 63.96 38.9697C64.8 38.9697 65.54 39.1964 66.18 39.6497C66.82 40.103 67.32 40.7497 67.68 41.5897C68.0267 42.4164 68.2 43.3964 68.2 44.5297ZM68 36.7725H53.9V34.2125H68V36.7725ZM68.2 24.3675C68.2 25.9275 67.9067 27.2475 67.32 28.3275C66.7333 29.3942 65.9 30.2075 64.82 30.7675C63.74 31.3275 62.46 31.6075 60.98 31.6075C59.4867 31.6075 58.2 31.3208 57.12 30.7475C56.0267 30.1742 55.1867 29.3475 54.6 28.2675C54 27.1875 53.7 25.8942 53.7 24.3875C53.7 23.4008 53.8467 22.4742 54.14 21.6075C54.42 20.7408 54.8133 20.0208 55.32 19.4475L57.32 20.3075C56.8267 20.9342 56.4733 21.5742 56.26 22.2275C56.0467 22.8808 55.94 23.6008 55.94 24.3875C55.94 25.8808 56.3733 27.0142 57.24 27.7875C58.1067 28.5475 59.3533 28.9275 60.98 28.9275C62.62 28.9275 63.8667 28.5342 64.72 27.7475C65.5733 26.9608 66 25.7942 66 24.2475C66 23.8075 65.9667 23.3608 65.9 22.9075C65.82 22.4408 65.7067 21.9808 65.56 21.5275H62.28V24.7675H60.38V19.3875H67.16C67.48 20.0408 67.7333 20.8142 67.92 21.7075C68.1067 22.6008 68.2 23.4875 68.2 24.3675ZM68 16.4209H53.9V14.5009L63.74 6.92094H53.9V4.54094H68V6.44094L58.12 14.0409H68V16.4209Z" fill="#003650"/>
				
					<path id="pipe-two" class="processPipe" d="M74.4998 1538.13V1.5H91.7731V1538.35C91.7731 1554.39 105.781 1567.14 120.937 1567.14H191V1587.05 H120.937C95.3446 1586.94 74.4998 1566.32 74.4998 1538.13Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
					<path id="pipe-two-one" class="processPipe" d="M98.9021 1454.5L98.9021 1460.44L66.4999 1460.44L66.4999 1454.5L98.9021 1454.5Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
					<path id="pipe-two-two" class="processPipe" d="M98.9021 936.869L98.9021 942.81L66.4999 942.81L66.4999 936.869L98.9021 936.869Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
					<path id="pipe-two-three" class="processPipe" d="M98.9021 367.066L98.9021 373.007L66.4999 373.007L66.4999 367.066L98.9021 367.066Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
					<path id="pipe-two-four" class="processPipe" d="M189.5 1560.5H194.989V1594.79H189.5V1560.5Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
				
					<path id="pipe-one" class="proteinPipe" d="M26.5 326.813V1.5H43.7733V327.037C43.7733 343.081 57.7816 355.829 72.9369 355.829H191.5V376.983H72.9369C47.3448 375.628 26.5 355.006 26.5 326.813Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
					<path id="pipe-one-one" class="proteinPipe" d="M19.5 307.44V301.5H51.9022V307.44H19.5Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
					<path id="pipe-one-two" class="proteinPipe" d="M190.5 349.5H195.989V383.787H190.5V349.5Z" fill="#5C889F" stroke="#C1D3DB" stroke-width="3"/>
				
					<circle id="circle-one" cx="215" cy="367" r="12" fill="#F79D5E"/>
					<circle id="circle-two" cx="215" cy="1577" r="12" fill="#F79D5E"/>
				</svg>				
			</div>

			<div id="contentcol" class="contentcol">
				<div class="overview">
					Our modelling work falls into two very different but complementary categories: process design and protein design. For protein design, we used protein modelling and molecular dynamics software to evaluate our rationally-designed copper-binding protein candidates. For process design, we use partial differential equation models to better understand our packed column reactor and optimise its design. 
				</div>

				<div class="protein">
					PROTEIN DESIGN
				</div>
				
				<div class='info'>
					<p>Several distinct modelling approaches were used as part of our rational protein design.</p>
					<h1>Rosetta</h1>
					<h2>Introduction to Rosetta</h2>
					<p>To analyze the proposed design changes from the rational design process, software tools were used to provide initial analysis and modelling of the structure. Among various computational protein design platforms, Rosetta was chosen for redesigning the structure and conducting energy analyses. The Rosetta software suite offers algorithms and protocols for macromolecular structure prediction and analysis of protein structures (Leaver-Fay et al., 2011). As a unified software package, it provides a wide range of tools suited for the project: to redesign the protein structure that has an increased binding affinity for Cu<sup>2+</sup> (Kaufmann, Lemmon, DeLuca, Sheehan & Meiler, 2010). </p>
					<p>Within the Rosetta software suite, the tasks and operations within the libraries are combined as algorithms called <b>protocols</b>. Some of protocols’ major functionalities include protein structure prediction, protein docking, protein design and energy scoring analysis. In this project, the structure of the original protein is known, therefore only energy scoring functions and the protein design protocols are used to assess how the residue-specific mutations increase stability and binding affinity (Leaver-Fay et al., 2011). Some specific functions such as Relax, a protocol for simple all-atom refinement of input structure, and Point Mutant Scan Application, a protocol for mutating the protein structure at specific residues and analyzing its stability, are used in the rational design process ("Applications for Macromolecule Design", 2020). </p>
					<h2>Protein Mutation Procedure</h2>
					<p>Using the Rosetta official documentation and the <a href="http://2016.igem.org/wiki/images/b/b6/TUE_Rosetta_beginners_guide.pdf">Rosetta Guide for the iGEM beginner</a> made by Team iGEM Technion and iGEM TU Eindhoven 2016, a structure analysis procedure was developed. The protocols mentioned in the following steps are written in a shell script. </p>
					<ol>
						<li>Obtain the original Protein Data Bank (PDB) file of the Mst-CopC structure. (It is accessible from RCSB PDB.)</li>
						<li>
							Prepare the structure.
							<ol>
								<li>Clean the PDB file while keeping the ligand. (This removes the information in PDB that Rosetta doesn’t use. (&quot;How to prepare structures for use in Rosetta&quot;, 2020))</li>
								<li>Run Relax protocol. (This reduces minor steric clashes in the protein-ligand complex. Running this protocol ensures the structure is at its most stable form before making mutations (&quot;Relax application&quot;, 2020). ) </li>
							</ol>
						</li>
						<li>Point mutant scan application. (Though the algorithm can scan for point mutants for given residues and determine the change in Gibbs free energy a mutation causes, a mutant file can be used to specify the mutants on specific residues determined in the rational design phase. )</li>
						<li>Energy scoring analysis for the overall structure and residue-specific energy breakdown. (Residue specific energy and overall energy information will be produced as outputs along with mutated PDB files in step 3. ) </li>
					</ol>
					<p>At step 2.2 Relax and step 3 Point Mutation, relaxed and mutated protein structures will be produced as PDB files along with the energy score of each output. The scoring function in Rosetta is an optimized energy function that calculates the energy of all atomic interactions in a globular protein (&quot;Scoring Tutorial&quot;, 2020). The score is a weighted sum of energy terms, including some representing physical forces like electrostatics and Van der Waal’s interactions and others statistical terms that indicate if the protein structure looks like any known structures (&quot;Scoring Tutorial&quot;, 2020). A complete list of energy terms and weights used in the scoring function can be found on the <a href="https://www.rosettacommons.org/demos/latest/tutorials/scoring/scoring">Rosetta Scoring Tutorial Documentation</a>. Therefore, instead of presenting the scores in physical energy units like kcal/mol, a custom Rosetta Energy Units (REU) is used due to the presence of the statistical terms. This score allows us to compare stability between scored structures. The lower the score, the more stable a structure is, which is sufficient to evaluate the relative stability of the mutated CopC-Cu<sup>2+</sup> candidates. </p>
					<h2>Results</h2>
					<h3>Single Residue Mutation</h3>
					<p>The table below shows the overall energy scores in REU and energy term breakdown for all protein candidates mutated on a single residue. The first column of the table indicates the mutation applied to the structure: 5icu is the PDB ID for Mst-CopC, and mutation code 	&lt;original amino acid&gt;&lt;residue number&gt;&lt;new amino acid&gt; follows the ID. </p>
					<p>As seen below, the original score of the protein is -320.377. The mutated proteins are sorted in ascending total scores, meaning that they are ordered in decreasing stability. Among all candidates, only the mutation from histidine to aspartic acid at residue 85 (H85D) produced a lower energy score than the original structure. However, most candidate’s scores are between -320 to -290, which means most candidates are relatively stable after mutation and molecular dynamic analysis can be done to further analyze the candidates. </p>
					<table>
						<thead>
							<tr>
								<th>description</th>
								<th>total_score</th>
								<th>description</th>
								<th>total_score</th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td>5icu</td>
								<td>-320.377</td>
								<td>5icu.L80D</td>
								<td>-310.078</td>
							</tr>
							<tr>
								<td>5icu.H85D</td>
								<td>-327.949</td>
								<td>5icu.L80V</td>
								<td>-308.632</td>
							</tr>
							<tr>
								<td>5icu.Y34F</td>
								<td>-319.826</td>
								<td>5icu.V79D</td>
								<td>-308.15</td>
							</tr>
							<tr>
								<td>5icu.F3H</td>
								<td>-319.02</td>
								<td>5icu.H85I</td>
								<td>-304.996</td>
							</tr>
							<tr>
								<td>5icu.S82E</td>
								<td>-318.491</td>
								<td>5icu.G29D</td>
								<td>-299.658</td>
							</tr>
							<tr>
								<td>5icu.V5K</td>
								<td>-318.478</td>
								<td>5icu.D83E</td>
								<td>-298.052</td>
							</tr>
							<tr>
								<td>5icu.I86E</td>
								<td>-318.333</td>
								<td>5icu.V79E</td>
								<td>-297.659</td>
							</tr>
							<tr>
								<td>5icu.I86K</td>
								<td>-317.939</td>
								<td>5icu.V79L</td>
								<td>-295.35</td>
							</tr>
							<tr>
								<td>5icu.I86H</td>
								<td>-317.353</td>
								<td>5icu.D83H</td>
								<td>-291.52</td>
							</tr>
							<tr>
								<td>5icu.H85E</td>
								<td>-317.217</td>
								<td>5icu.D83G</td>
								<td>-291.469</td>
							</tr>
							<tr>
								<td>5icu.I86D</td>
								<td>-316.911</td>
								<td>5icu.V79H</td>
								<td>-273.441</td>
							</tr>
							<tr>
								<td>5icu.V87T</td>
								<td>-314.902</td>
								<td>5icu.S2D</td>
								<td>-270.366</td>
							</tr>
							<tr>
								<td>5icu.G28D</td>
								<td>-313.721</td>
								<td>5icu.S81H</td>
								<td>-264.303</td>
							</tr>
							<tr>
								<td>5icu.L80E</td>
								<td>-313.472</td>
								<td>5icu.G29H</td>
								<td>-261.843</td>
							</tr>
							<tr>
								<td>5icu.S81D</td>
								<td>-312.271</td>
								<td>5icu.S81E</td>
								<td>-259.4</td>
							</tr>
							<tr>
								<td>5icu.R78H</td>
								<td>-312.167</td>
								<td>5icu.G29E</td>
								<td>-256.144</td>
							</tr>
							<tr>
								<td>5icu.L80H</td>
								<td>-311.984</td>
								<td>5icu.S81V</td>
								<td>-246.804</td>
							</tr>
							<tr>
								<td>5icu.G27S</td>
								<td>-311.112</td>
								<td>5icu.G29V</td>
								<td>-216.733</td>
							</tr>
							<tr>
								<td>5icu.G28H</td>
								<td>-311.068</td>
								<td>5icu.S2H</td>
								<td>-137.613</td>
							</tr>
							<tr>
								<td>5icu.G28E</td>
								<td>-311.022</td>
								<td>5icu.S2E</td>
								<td>-99.242</td>
							</tr>
						</tbody>
					</table>
					<div class="caption">Table 1. Rosetta energy scores for the original Mst-CopC “5icu” and mutated candidates.</div>
					<p>The complete table of results is found <a href="https://2020.igem.org/wiki/images/5/55/T--Waterloo--Rosetta_Table.pdf">here</a></p>
					<p>These PDB files also contain per-residue energy scores, which can be used to compare the effect each mutation has on each residue. We assembled these scores into heatmaps, where each row corresponds to either a mutation or the original protein, and each column corresponds to a certain residue. This data was difficult to visualize effectively due to the presence of outliers, so certain data transformations were applied to better analyze some aspect of our energy score results. (The heatmap can be found in our github as heatmap_single_relative.png.)</p>		
					<img src="https://2020.igem.org/wiki/images/8/8c/T--Waterloo--heatmap-single-relative.png" alt="Figure 1. Heat map with unmutated protein energy scores subtracted.">
					<div class="caption">Figure 1. Heat map with unmutated protein energy scores subtracted. </div>
					<p>One such transformation is the above heatmap, which has subtracted the unmutated proteins energy scores from each mutant (and itself). The resulting scores are therefore relative to the unmutated protein, and show increases or decreases in score, aiding one in finding whether a mutation is beneficial to the energy score of each residue.</p>
					<img src="https://2020.igem.org/wiki/images/e/e1/T--Waterloo--heatmap-single-scaled.png" alt="Figure 2. Heat map with each column scaled to itself.">
					<div class="caption">Figure 2. Heat map with each column scaled to itself.</div>
					<p>Seen above is another heatmap, except that each column has been scaled to the range [0, 1], highlighting changes in energy score. Unlike the previous heatmap, it does not indicate the scale of these changes.</p>
					<p>Note that in both heatmaps the rows and columns have the same labels, and represent the same things, only the values contained within the map itself have been modified. </p>
					<p>Using these heatmaps, it is possible to rationally select mutations to simultaneously apply in order to further reduce the overall energy scores. Possible next steps include applying a logarithmic scale to better differentiate between the values in the heatmap.</p>
					<h3>Double Residue Mutations</h3>
					<p>To determine a suitable combination of two mutations, it is computationally feasible to simply run all possible pairs of mutations from our set of rationally selected single mutations. The overall energy scores of each of these “double mutants” was also assembled into a heatmap, as an efficient method of comparing all of these mutants at once.</p>
					<img src="https://2020.igem.org/wiki/images/5/5d/T--Waterloo--heatmap-double.png" alt="Figure 3. Heat map of double residue mutations.">
					<div class="caption">Figure 3. Heat map of double residue mutations.</div>
					<p>Each row and column respond to a particular mutation, with each intersection representing both mutations applied together. The symmetry appears to be inexact, a phenomenon which was likely due to how Rosetta applies multiple mutations.</p>
					<p>Based on the results from Rosetta and analyzing heatmaps, a list of optimal candidates is compiled. These mutated candidates will be further assessed using more detailed molecular dynamic analysis. </p>
					<p>The list of mutations passed onto MD: </p>
					<ul>
						<li>D83E</li>
						<li>F3H</li>
						<li>G28E</li>
						<li>H85D</li>
						<li>I86K</li>
						<li>S82E</li>
						<li>S82E, F3H</li>
						<li>F3H, V87T</li>
						<li>H85D, S82E</li>
						<li>Y34F, F3H, S81D</li>
						<li>H85D, V87T, I86K, G28E</li>
						<li>Y34F, F3H, S81D, H85D</li>
					</ul>
					<h1>Molecular Dynamics</h1>
					<p>Molecular dynamics was used to test the relative binding affinities of these candidate proteins. We used GROMACS (Berendsen et al., 1999) along with the Charmm36 (Vanommeslaeghe et al., 2010) force field parameters which include parameters for copper ions in solution. This allows us to estimate the total interaction energy between the copper ion and the protein, and using the relative magnitudes of these interaction energies allows us to compare the strength of binding. Molecular dynamics provides a more rigorous method for comparing binding strength due to the fact that a molecular dynamics simulation takes into account the motion of every individual atom in the system and their interactions.</p>
					<p>All GROMACS simulations were run on the Graham computing cluster located at the University of Waterloo and hosted by ComputeCanada.</p>
					<p>In order to implement a molecular mechanics simulation, the protein pdb files were converted to the gromacs format using Charmm-GUI, an online tool developed in order to construct topology files and parameter files for proteins using the Charmm force field. Once the required files were attained, the system was placed in solution in a process known as solvation, and the charge in the system was balanced by introducing faraway sodium or chloride ions. The system then undergoes an energy minimization process in which the protein and copper ion are restrained while the solvent molecules settle into place. This avoids extreme behaviour that could occur due to collisions between molecules. The thermodynamics of the system is then set up using standard NVT and NPT equilibration procedures in GROMACS, and then the final simulation is run. </p>
					<p>A typical ten nanosecond simulation was performed for each candidate protein. The results are found in the table below.</p>
					<table>
						<thead>
							<tr>
									<th>Name</th>
									<th>Total Interaction Energy</th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td>Mst-CopC</td>
								<td>-540.6373 kJ/mol</td>
							</tr>
							<tr>
								<td>H85D</td>
								<td>-1278.448 kJ/mol</td>
							</tr>
							<tr>
								<td>F3H</td>
								<td>-563.9996 kJ/mol</td>
							</tr>
							<tr>
								<td>G28E</td>
								<td>-607.4388 kJ/mol</td>
							</tr>
							<tr>
								<td>I86K</td>
								<td>-586.829 kJ/mol</td>
							</tr>
							<tr>
								<td>S82E</td>
								<td>-588.4093 kJ/mol</td>
							</tr>
							<tr>
								<td>D83E-V87T</td>
								<td>-853.725 kJ/mol</td>
							</tr>
							<tr>
								<td>H85D-S82E</td>
								<td>-1392.7281 kJ/mol</td>
							</tr>
							<tr>
								<td>H85D-V87T-I86K-G28E</td>
								<td>-1246.7286 kJ/mol</td>
							</tr>
							<tr>
								<td>S82E-F3H</td>
								<td>-557.0123 kJ/mol</td>
							</tr>
							<tr>
								<td>Y34F-F3H-S81D</td>
								<td>-1145.2119 kJ/mol</td>
							</tr>
							<tr>
								<td>Y34F-F3H-S81D-H85D</td>
								<td>-1698.6439 kJ/mol</td>
							</tr>
						</tbody>
					</table>
					<div class="caption">Table 2. Molecular dynamics interaction energies for the original Mst-CopC and selected mutation candidates. </div>
					<p>From the table of results we can see that candidate Y34F-F3H-S81D-H85D has the highest relative binding affinity when compared to Mst-CopC, with a 3.14 improvement in total interaction energy over the original Mst-CopC. Figure 4 shows the copper ion in the H85D mutant’s binding site. This animation was produced using VMD, a free software tool for visualizing molecular dynamics simulations.</p>
					<img src="https://2020.igem.org/wiki/images/1/13/T--Waterloo--GROMACS_GIF.gif" alt="Animation of Cu2+ in H85D binding site">
					<div class="caption">Figure 4. Molecular dynamics animation of Cu<sup>2+</sup> (bronze coloured sphere) in the binding site of the Mst-CopC H85D mutant.</div>
				</div>

				<div class="process">
					PROCESS DESIGN
				</div>

				<div class='info'>
					<h1>Motivation For Our Treatment Process</h1>
					<p>Our metal-removal technology will compete with a variety of other aqueous metal waste treatment processes. Some current methods for treating aqueous metal wastes include chemical precipitation, ion exchange, adsorption, and membrane filtration (Fu et al., 2011). Chemical precipitation is particularly widespread because of its relatively simplicity and low cost (Fu et al., 2011). In this method, the pH of the metal waste is increased by the addition of a base such as lime, causing the metal ions to precipitate as insoluble hydroxides that can be recovered as solid sludge. However, removing water from this sludge can be costly, and the resulting sludge is typically disposed of, so valuable metals cannot be recovered or reused. Ion exchange, membrane filtration, and membrane filtration are all relatively new technologies (Fu et al., 2011), so high costs often prohibit large scale applications. Furthermore, there are few established methods that recover metal ions from industrial wastewaters to allow recycling. We therefore aim to fill in this gap in metal waste treatment by using metal binding proteins to create a novel metal removal and recovery process. This process should quickly, selectively, and cheaply remove metal ions from wastewater, allowing metal ions to be recovered and recycled, providing economic and environmental benefits. </p>
					<p>In order for our designed metal and cellulose binding fusion protein to have merit in the real world, it is imperative to design a suitable industrial process that can use our biotechnology to treat metal waste. </p>
					<p>We will design a reactor to treat copper-containing metal finishing waste, at a scale relevant for a typical semiconductor manufacturing plant. Metal finishing wastes result from processes including electroplating, in which dissolved metals (copper, in this case) are deposited in thin layers onto solid surfaces using electrochemical reactions (Arán, 2017). After the process is complete, the remaining solutions must be treated before disposal. However, since dissolved metal wastes are produced by a large variety of industrial processes, the design and modelling process applied here could be used for any other waste stream and dissolve metal(s). </p>
					<h1>Choosing Our Reactor Type</h1>
					<p>In order to effectively treat large volumes of aqueous waste, we immobilised our cellulose and metal binding fusion protein inside an industrial bioreactor. Some advantages granted by using a bioreactor include: reactor size control and dimension tunability, continuous operation, and a wide variety of sub-types of reactors to base our design off of (Shuler &amp; Kargi, 2002). Furthermore, the bioreactors have long been used for industrial scale processes that operate based on enzymatic or cellular function, so research in this area is well developed. </p>
					<p>Of the many varieties of bioreactors available, we chose to use a packed column reactor. A packed column reactor consists of a large container filled with some sort of porous packing material, and effluent is fed through the bed, allowing a reactor to occur (Shuler &amp; Kargi 2002). For our reactor, we will use cellulose beads as a packing material. Packed column reactors enable large volumes of waste to be treated in a continuous manner, providing much greater efficiency than batch processes that treat a fixed volume of waste at one time (Shuler &amp; Kargi 2002). For our metal removal application, all removed metal will stay trapped inside the column as waste is fed through. Thus, the metal can be removed at a convenient time, such as during off-peak waste production hours.</p>
					<p>Since our fusion proteins include a cellulose binding domain that binds cellulose very strongly, the fusion proteins can be immobilised on the cellulose packing by simply feeding them into the column. By immobilising our fusion proteins on the surface of the cellulose packing, we are able to maximize the surface area of reactive sites, thus increasing the treatment rate of the wastewater system. </p>
					<p>With this general design in mind, we set out to use engineering and modelling tools to decide how to build a suitable reactor to treat our waste stream of interest.</p>
					<p>We envision that our reactor will operate in two stages: </p>
					<ol>
						<li>First, the metal containing wastewater will be flowed through the reactor, allowing the metal in the wastewater to bind to the proteins within. This will produce a stream of clean water that can be discharged to the environment. This is the <b>“binding”</b> stage.</li>
						<li>Secondly, once the reactor is saturated with metal, we will rinse the reactor with another solution of a different pH, which will cause the metal ions to unbind from the reactor. This will result in a concentrated and pure solution of dissolved metal. This resulting solution could then be recycled for various applications. This is the <b>“elution”</b> stage.</li>
					</ol>
					<p>Below is a 3D model that provides an example of our reactor’s general construction. Wastewater will be pumped into the reactor from the bottom, passing through the cellulose packing before exiting through the top of the reactor. </p>
					<img src="https://2020.igem.org/wiki/images/a/a0/T--Waterloo--Reactor_Column_Design.jpeg" alt="3D Model of our reactor design">
					<div class="caption">Figure 5. 3D model of our packed column reactor created using CAD.</div>
					<h1>Developing Equations For Modelling</h1>
					<p>In order to model a system undergoing chemical reactions, we must first develop a kinetic rate law for the reactions. For our system, we will use a straightforward dimensional analysis approach similar to the Langmuir isotherm. This is a commonly applied model for adsorption of a reactant onto a protein. </p>
					<p>To determine the net rate of adsorption of Cu<sup>2+</sup> onto CopC, we add the forward binding rate to the reverse binding rate. A key nuance of this model is that the concentration of CopC is given as a surface concentration. In order to relate the surface concentration to the rate of change of a volumetric concentration, a conversion factor is computed by considering the surface area per volume of BMCC cellulose that the protein is attached to.</p>
					<p>Suppose the surface area per volume of BMCC is given by a parameter \( a \) and that the fraction of the reactor occupied by cellulose is \( f \). Let \( m = [Cu^{2+}] \), let \( Q \) be the initial protein surface concentration, and let \( q = Q - [CopC] \). </p>
					<p>Free protein + Free metal \( \underset{k_d}{\stackrel{k_a}{\rightleftharpoons}} \) </p>
					<ul class="noBullet">
						<li>Protein-metal complex</li>
						<li>\( k_a \): adsorption rate constant, \( k_d \): desorption rate constant</li>
					</ul>
					<p>Adsorption (forward) rate:</p>
					<p>\( v_a = k_a  m  N \)</p>
					<ul class="noBullet">
						<li>Where \( N = (Q - q) \)</li>
						<li>So, \( v_a = k_a m (Q - q) \)</li>
					</ul>
					<p>Desorption (reverse) rate:</p>
					<p> \( v_d = k_d q \)</p>
					<ul class="noBullet">
						<li>Variables:</li>
						<li>\(  v \): rate (\( \frac{mol}{m^2  s} \))</li>
						<li>\( m \): free metal concentration (\( \frac{mol}{L} \))</li>
						<li>\( k_a \): adsorption rate constant (\( \frac{L}{mol s} \))</li>
						<li>\( k_d \): desorption rate constant (\( \frac{1}{s} \))</li>
						<li>\( N \): free sites (\( \frac{mol}{m s} \))</li>
						<li>\( Q \): total sites (\( \frac{mol}{m s} \))</li>
						<li>\( q \): occupied sites (\( \frac{mol}{m s} \))</li>
					</ul>	
					<p>Net rate:</p>
					<p>\( \frac{dq}{dt} = v_a - v_d = k_a m (Q - q) - k_d q \)</p>
					<p>\( \frac{dq}{dt} \) is a rate in \(\frac{mol}{m^2 s} \) (this is how much metal adsorbs to a given surface area each second)</p>
					<p>But, we’re interested in the change \(\frac{dm}{dt} \), i.e. the change in metal concentration per second</p>
					<p>For a given volume in the packed bed \( V \), a portion is liquid (\( V_{aq} \)) and a portion is packing material (\( V_p \))</p>
					<p>There is also a specific surface area, i.e., the packing surface area (\(s \)), per total volume (\( V \))</p>
					<p>Define specific surface area \(\rho = \frac{s}{V} \left( \frac{m^2}{m^3} = \frac{1}{m} \right) \)</p>
					<ul class="noBullet">
						<li>And void fraction \( f = \frac{V_{aq}}{V} \left( \frac{m^3}{m^3} = 1 \right) \)</li>
						<li>To convert \( \frac{dq}{dt} \) to \( \frac{dm}{dt} \),</li>
					</ul>
					<p>\(  \frac{dm}{dt} = -\frac{dq}{dt} \left( \frac{s}{V} \right) \left( \frac{V}{V_{eq}} \right) = -\frac{dq}{dt} \frac{\rho}{f} \)</p>
					<p>So, \( \frac{dm}{dt} = -\frac{\rho}{f} \left( k_a m (Q - q) - k_d q \right) \)</p>
					<p>Then the net adsorption rate is given by</p>
					<p>\( \frac{dm}{dt} = -(a/f) (k_a m (Q-q) - k_d q) \)</p>
					<p>Using this model allows computation of important properties of well-mixed reactors, such as a batch reactor where there is no inflowing or outflowing material. For a nominal input concentration of Cu<sup>2+</sup> solution the results and a list of relevant parameters can be found below. </p>					
					<p>However, we aim to design a packed column reactor with a constant flow of new material, requiring us to consider transport phenomena. In order to incorporate transport phenomena this model needs to be extended to a partial differential equation. This can be done by employing the conservation of mass law as well as a constitutive equation called Fick’s law which governs the rate of diffusion of solute in the reactor. These transport-reaction models are described in the Encyclopaedia of Systems Biology published by Springer (Clairambault, 2013).</p>
					<p>To begin deriving the transport equation, one must first write down the governing conservation and constitutive relations. The conservation of mass equation for a three dimensional reactor of fixed volume \( V \) is given by the following equation, where \( \partial V_{\pm} \) denotes the inflow (+) and outflow (-) boundaries of the reactor. </p>
					<p class="eqn">\( \frac{d}{dt} \iiint_V m(x,y,z) dxdydz =  \Phi(x,y)|_{\partial V_+} - \Phi(x,y)|_{\partial V_-} + \iiint_V R(x,y,z) dxdydz \)</p>
					<p>Here \( R \) is the reaction rate described by the kinetic model. The flux terms \( \Phi \) can be computed by combining the advective flux and the diffusive flux. The advective term describes mass travelling through the reactor at a velocity \( \vec{v} \) (which can be a function of position), and can be written as \( \Phi = \vec{v} m \). The diffusive term describes mass entering and exiting the reactor due to the concentration gradient, which is described by Fick’s Law:</p>
					<p class="eqn"> \( \Phi_{diffusion} = D \frac{dm}{dz} \)</p>
					<p>This yields the total mass conservation equation:</p>
					<p class="eqn">\( \frac{d}{dt} \iiint_V m(x,y,z) dxdydz \) </p>
					<p class="eqn">\( = -\vec{v}(m(x,y,z_{+}) - m(x,y,z_{-})) + D(\frac{dm}{dz}(x,y,z_+) - \frac{dm}{dz}(x,y,z_-)) + \iiint_V R(x,y,z) dxdydz \)</p>
					<p>Here we may apply the fundamental theorem of calculus to convert the flux terms into an integral over \( \partial V \).</p>
					<p class="eqn">\( \frac{d}{dt} \iiint_V m(x,y,z) dxdydz - \iiint_V R(x,y,z) dxdydz = \iint_{\partial V} -m \vec{v}\cdot \hat{n} + D\nabla m \cdot \hat{n} dxdy \)</p>
					<p>We can apply Gauss’s Divergence Theorem in order to convert the double integrals into triple integrals.</p>
					<p class="eqn">\( \frac{d}{dt} \iiint_V m(x,y,z) dxdydz = \iiint_{V} R(x,y,z) -\nabla(m) \cdot \vec{v} + D\nabla^2 m  dxdydz \)</p>
					<p>And finally, by applying the Reymond-Dubois lemma we will arrive at the following system of partial differential equations.</p>
					<p class="eqn">\( \frac{dm}{dt} + \vec{v}\cdot \nabla m - D\nabla^2 m = -(\frac{a}{f}) (k_a m (Q-q) - k_d q) \)</p>
					<p class="eqn">\( \frac{dq}{dt} = k_a m (Q-q) - k_d q \)</p>
					<p>Unfortunately due to numerical problems, a solution to this system has not been completely developed. More work on improving numerical stability must be done before this will produce significant results. The main issue faced in solving this model using the finite element method is the timescale discrepancy between the transport and the reaction rate. From our review of the literature it appears that a typical reaction rate for our system will be much faster than the speed at which solution is transported by the advective term in the above equation. This implies that a very high resolution simulation must be performed, and advanced computational techniques may be required. However, we found that transport effects may be neglected locally in the reactor by a quasi-steady state approximation which is explained below. This allows us to still compute meaningful results from the model despite not having an exact solution.</p>
					<p>Once a reactor has been designed, we can use the Ergun equation (Ergun, 1949), a standard tool in chemical engineering, to determine the pressure drop across the packed column. Pressure drop corresponds to the positive pressure required at the input end of the reactor to drive fluid flow through the reactor. This positive pressure would need to be applied using a pump. An approximate version of the Ergun equation suitable for a liquid packed column is listed below (Wu, 2007). All parameters necessary for computation of this pressure drop can be found in the parameter list above and the description of the reactor.</p>
					<p class="eqn">\( \frac{\Delta P}{L} = \frac{72 \mu \tau (1 - \epsilon)^2 v_s}{D^2_P \epsilon^3} + \frac{3 \tau (1 - \epsilon) \rho v^2_s \left(\frac{3}{2} + \frac{1}{\beta^4} - \frac{5}{2 \beta^2} \right)}{4 \epsilon^3 D_p} \)</p>
					<p>Where L is the height of the reactor, and tau and beta are the tortuosity and pore-to-throat ratio which are defined as follows.</p>
					<p class="eqn">\(  \tau = \frac{1}{2}\left[1 +\frac{1}{2} \sqrt{1 - \epsilon} +\sqrt{1 - \epsilon}\frac{\sqrt{\left(\frac{1}{\sqrt{1 - \epsilon}} - 1\right)^2 + \frac{1}{4}}}{1 - \sqrt{1 - \epsilon}}\right] \)</p>
					<p class="eqn">\( \beta = \frac{1}{1 - \sqrt{1 - \epsilon}} \)</p>
					<h1>Computational Approaches</h1>
					<p>To simulate the partial differential equation model, we decided to use <a href="https://fenicsproject.org">FEniCS</a>, a computing platform which solves partial differential equations and has an easy-to-use Python interface. To use FEniCS in a computational environment on Windows (used by most of us), it needs to be installed on the <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10">Windows Subsystem for Linux</a> (WSL). To use FEniCS on macOS, the macOS Anaconda package can be used. For editing while using WSL, we found <a href="https://code.visualstudio.com">VS Code</a> with the <a href="https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl">Remote - WSL extension</a> to be most useful. Last, for source code management, we used a <a href="https://github.com/igem-waterloo/uwaterloo-igem-2020">Git repository</a> for the team. This environment set-up allowed us to work efficiently and collaborate easily. Our main FEniCS simulation code can be found <a href="https://github.com/igem-waterloo/uwaterloo-igem-2020/blob/master/Fenics/reaction_system.py">here</a>.</p>
					<h1>Getting Parameters</h1>
					<p>Once model equations had been developed, it was necessary to find values for the parameters so that useful results could be computed. These parameters were found directly in literature, or computed from combinations of other parameters. </p>
					<p>Parameters are required to describe the scale of the problem that we are trying to address using this reactor. We need to know the volume of waste to be treated per day and the concentration of dissolved copper in that waste. We also need to know the acceptable concentration of dissolved metal in the treatment outflow, so that it can be safely released to the environment. </p>
					<p>To estimate the amount of copper-containing waste to treat each day, we looked to existing electronics manufacturers that produce metal-finishing wastes. One semiconductor manufacturer in Taiwan reported producing 4.44 million metric tonnes of dissolved metal waste over three manufacturing facilities each year (“TSMC Corporate Social Responsibility Report”, 2019). This would correspond to 1.48 million metric tonnes per year per facility, or about 46.9 L/s at each facility. For this reason, we will design a reactor that can treat an average of 46.9 L/s of waste over the course of each day.  </p>
					<p>Chiu et al. (1987) states that typical copper concentrations in metal finishing wastewaters can range from 1-30 mg/L. For this reason, we will design our reactor to be able to treat 30 mg/L dissolved copper. </p>
					<p>The United States Environmental Protection Agency (USEPA) states that the permissible limit for dissolved copper in industrial wastes discharged to the environment is 1.3 mg/L (Al-Saydeh et al., 2017). For this reason, our reactor must be able to reduce the concentration of dissolved copper from 30 mg/L to below 1.3 mg/L. </p>
					<p>The parameters relevant to the cellulose binding domain and the cellulose packing material are the specific surface area and the surface protein concentration. These were found in McLean (2000) and are listed in the table below. The fraction of volume occupied by cellulose for a typical 3 millimeter diameter packing material is found in (Sen, 2015).</p>
					<p>Similarly, an equilibrium constant for a variant of CopC was found in Young et al (2015) and a forwards rate constant for a structurally and functionally similar copper binding protein was found in Du, 2013. The equilibrium constant is necessary to determine whether the reaction happens at an appreciable rate relative to the fluid transport speed in the reactor. This rate constant provides an estimate of the reaction timescale which will inform how large our reactor must be.</p>
					<table>
						<thead>
							<tr>
								<th>Description</th>
								<th>Variable</th>
								<th>Value</th>
								<th>Source</th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td>Input Concentration (Cu)</td>
								<td>\(m\)</td>
								<td>30 \( \frac{mg}{L}\)</td>
								<td>Chiu, 1984</td>
							</tr>
							<tr>
								<td>Output Concentration (Cu)</td>
								<td></td>
								<td>1.3 \( \frac{mg}{L}\)</td>
								<td>Saydeh et al, 2017</td>
							</tr>
							<tr>
								<td>Surface Protein Concentration</td>
								<td>\(Q\)</td>
								<td>0.35E-6 \( \frac{mol}{m^2}\)</td>
								<td>McLean, 2000</td>
							</tr>
							<tr>
								<td>Specific Surface Area</td>
								<td>\(a\)</td>
								<td>72600000 \(\frac{1}{m}\)</td>
								<td>McLean, 2000</td>
							</tr>
							<tr>
								<td>Forwards Rate Constant</td>
								<td>\(k_a\)</td>
								<td>0.015 \(\frac{1}{s}\)</td>
								<td>Du, 2013</td>
							</tr>
							<tr>
								<td>Reverse Rate Constant</td>
								<td>\(k_d\)</td>
								<td>3e-16 \(\frac{1}{s}\)</td>
								<td>Du, 2013 and Young et al, 2015</td>
							</tr>
							<tr>
								<td>Equilibrium Constant</td>
								<td>\(K_D\)</td>
								<td>1E-13.7</td>
								<td>Young et al, 2015</td>
							</tr>
							<tr>
								<td>Void Fraction</td>
								<td>\(\rho\)</td>
								<td>0.31</td>
								<td>Sen, 2016</td>
							</tr>
							<tr>
								<td>Density of Cellulose</td>
								<td></td>
								<td>1.5 \(\frac{g}{cm^3}\)</td>
								<td>National Center for Biotechnology Information, 2020</td>
							</tr>
							<tr>
								<td>Porosity</td>
								<td>\(\epsilon\)</td>
								<td>0.69</td>
								<td>Sen, 2016</td>
							</tr>
							<tr>
								<td>Superficial Reactor Velocity</td>
								<td>\(v_s\)</td>
								<td>0.2552</td>
								<td>Computed based on reactor size. See below</td>
							</tr>
						</tbody>
					</table>
					<div class="caption">Table 3. Parameters, variables, values, and literature sources for each of the parameters used in our model’s equations. </div>
					<h1>Results & Discussion</h1>
					<p>The following plots are taken from the aforementioned model with the approximation that advective transport phenomena can be neglected when compared to the reaction timescale. This is reasonable because the copper binding protein chosen has a very small equilibrium constant, which implies that the reaction time scale is very fast. This model assumes that the reactor is well mixed, since the cellulose in the reactor will produce very strong eddy diffusion.</p>
					<img src="https://2020.igem.org/wiki/images/6/6c/T--Waterloo--Model_batch_reactor.png" alt="Batch Reactor Copper Concentration">
					<div class="caption">Figure 6: Concentration of copper in a batch reactor versus time. Note how the majority of the copper in the solution has been bound within 5 seconds.</div>
					<img src="https://2020.igem.org/wiki/images/c/c2/T--Waterloo--Model_batch_reactor_occupied.png" alt="Batch Reactor Occupied Protein">
					<div class="caption">Figure 7: Percentage of occupied protein active sites in batch reactor versus time, assuming one gram of cellulose packing material uniformly distributed throughout the batch reactor.</div>
					<p>From these two plots we can see that within 5 seconds a vast majority of the copper present in solution has adsorbed onto the CopC binding sites, while only a small fraction of the CopC has actually been used up. This implies that a packed column reactor assuming similar initial inflow concentrations will not saturate too quickly to be of use.</p>
					<p>It is also visible from the model that the timescale of the reaction is much smaller than the time a copper ion spends in a typical large packed column reactor. This led to numerical problems when solving the partial differential equation model, which we did not have time to fix. Instead, due to the rapid equilibrium approximation we may use the batch reactor to estimate a typical length scale for the reactor.</p>
					<p>Through some dimensional analysis, we can find that 99.5% of copper ions which spend longer than 5 seconds in the reactor will be adsorbed. This means that we can potentially treat 200 times the volume of the reactor in copper solution before the reactor is saturated. This approximation allows us to choose a scale for our reactor. </p>
					<p>For the metal finishing waste stream chosen, we will need to treat an average of 46.9 L/s of waste over the course of each day (see “Getting Parameters”), or 4 052 160 L/day. We choose to treat this waste using four 5-hour reactor cycles, allowing for 1-hour breaks in between for elution and other routine maintenance. This means that 1 013 040 L of waste must be treated in each of the four reactor cycles. Since we can treat 200 times the volume of the reactor in each reactor cycle, our reactor must be able to accommodate 1/200th of the waste per cycle, or 5065 L. However, due to the packing of 3 mm cellulose particles,, only 31% of our total reactor volume is actually free space. This means that we need the total volume of our packed column to be 16 339 L to accommodate 5065 L of liquid. Based on a 1:10 height-diameter ratio, this gives a packed column 1.28 m in diameter and 12.8 m in length.I n order to fill the reactor, we would require approximately 17 000kg of cellulose packing material, assuming a 3 mm particle diameter and 31 % void space. </p>
					<p>In our <a href="2020.igem.org/Team:Waterloo/Human_Practices">discussion with stakeholders from Aevitas Hazardous Management</a>, they told us that they used an approximately 30 000 L batch reactor to treat their wastes. Although we are using a continuous process with some differences from Aevitas’ process, this confirms that our proposed reactor scale of about 16 000 L is reasonable and within the size range of existing waste treatment processes.  </p>
					<p>With the above assumptions for our reactor, according to the estimate of 99.5% of all copper ions being depleted after a reactor cycle the effluent concentration should be below 1 µmol/L. This falls well below our goal of an output concentration of 1.3mg/L of copper.</p>
					<p>Now that the reactor has been designed, we can use the Ergun equation to compute pressure drop. The velocity of fluid in the reactor should then be chosen to be the height divided by 50 seconds, in order to allow the copper ions to stay in the reactor for longer than 5 seconds at a time. This means that the superficial flow velocity in the reactor is 0.2556 m/s. We apply the Ergun equation (Ergun, 1949), a standard tool in chemical engineering, to calculate the resulting pressure drop across our packed column reactor.</p>
					<p>For our reactor of diameter 1.28 m and length 12.8 m, the pressure drop is 0.126 bar. This means that in order for wastewater to flow through the reactor, we must use a pump to pressurize the reactor inlet to at least 0.126 bar greater than the outlet. This is very reasonable by industrial standards.</p>
					<h1>Next Steps & Implementation </h1>
					<p>Refer to the <a href="2020.igem.org/Team:Waterloo/Implementation">Implementation page</a> for a more detailed discussion of our envisioned implementation.</p>
					<p>The results of our modelling allow us to propose a reactor design that is suitable for treating a copper containing waste stream from a typical semiconductor manufacturing facility. Although this model provides the reactor scale and dimensions needed, there are several other parameters required in order to operate the reactor. </p>
					<p>In particular, we must determine the effect of pH on copper binding in the reactor. pH must be determined for both the initial binding phase and the subsequent elution phase. This would ideally be determined using experiments to characterize our protein’s copper binding at a range of pH values. However, information from literature can be used to make rough estimates of the pH for both the initial binding phase and the elution phase.</p>
					<p>In the binding stage, we want the copper to bind to the copper binding proteins in the reactor as strongly as possible. In CopC, the copper ion is coordinated by histidine, aspartic acid, and glutamic acid (Lawton, 2016). In order for copper binding to occur, these amino acids must all exist in their deprotonated forms so that their lone electron pairs can bind to copper. These deprotonated forms are also negatively or neutrally charged, attracting the positively charged copper ion. The deprotonated state can be achieved for all of these amino acids at a pH of 7.2 or higher (Včeláková et al., 2004). This means that the dissolved copper waste fed into the reactor must be at pH 7.2 or higher. Metal finishing waste is generally near neutral pH (van Dijken et al., 1999) so only small adjustments would be needed. The pH could be adjusted by adding small amounts of concentrated acid or base to the dissolved copper waste before feeding it to the reactor. </p>
					<p>In the elution stage, we want copper ions to bind to the copper binding proteins in the reactor as weakly as possible, so that the copper ions can be removed from the reactor. To weaken copper binding, we can protonate the histidine, aspartic acid, and glutamic acid residues involved in copper binding. Protonation prevents these amino acids’ lone pairs from bonding with the copper ion, and also causes the side chains to become neutrally or positively charged, further repelling the copper ion. Protonation of these amino acids can be achieved at pH 4.8 or lower (Včeláková et al., 2004). This means that metal ions can be removed from the reactor by flowing through a solution with a pH of 4.8 or lower. This would result in a concentrated and pure copper solution. If necessary, the pH of the recovered metal solution could be further adjusted for recycling by using small volumes of concentrated acid or base.</p>
					<p>Determining the pH required for the elution and binding stages of our process therefore moves us one step closer to building a functioning industrial process. The pH values determined herein are specific to copper and the CopC protein, but required pH values could be calculated for any metal and metal binding protein. In general, amino acids must be deprotonated to bind metals, so the metal binding step will always require a higher pH than the elution step. </p>
					<p>Below is a proposed schematic of our packed column reactor’s proposed implementation, explained further on the <a href="https://2020.igem.org/Team:Waterloo/Implementation">Implementation page</a>.</p>
					<img src="https://2020.igem.org/wiki/images/4/4c/T--Waterloo--process_imp.png" alt="A schematic of our packed column reactor’s proposed implementation and operation schedule.">
					<div class="caption">Figure 8. A schematic of our packed column reactor’s proposed implementation and operation schedule.</div>
				</div>
				<div class="refs">
					<p class="referencesHead">References</p>
					<p class="references">Clairambault, J. (2013). Reaction-Diffusion-Advection Equation. In W. Dubitzky, O. Wolkenhauer, K.-H. Cho, & H. Yokota (Eds.), Encyclopedia of Systems Biology (p. 1817). Springer New York. https://doi.org/10.1007/978-1-4419-9863-7_697</p>
					<p class="references">Fu, Fenglian, and Qi Wang. "Removal of heavy metal ions from wastewaters: a review." Journal of environmental management 92.3 (2011): 407-418.</p>
					<p class="references">Malik, et. al. "Detection and removal of heavy metal ions: a review." Environmental Chemistry Letters 17.4 (2019): 1495-1521. https://doi.org/10.1007/s10311-019-00891-z</p>
					<p class="references">Zoroddu, M. A., Medici, S., & Peana, M. (2009). Copper and nickel binding in multi-histidinic	peptide fragments. Journal of Inorganic Biochemistry, 103(9), 1214-1220. doi:10.1016/J.jinorgbio.2009.06.008</p>
					<p class="references">Včeláková, K., Zusková, I., Kenndler, E., & Gaš, B. (2004). Determination of cationic mobilities and pKa values of 22 amino acids by capillary zone electrophoresis. Electrophoresis, 25(2), 309-317. doi:10.1002/elps.200305751</p>
					<p class="references">Wijekoon, C. J., Young, T. R., Wedd, A. G., & Xiao, Z. (2015). CopC protein from Pseudomonas fluorescens SBW25 features a conserved novel high-affinity Cu<sup>2+</sup> binding site. Inorganic chemistry, 54(6), 2950–2959. https://doi.org/10.1021/acs.inorgchem.5b00031</p>
					<p class="references">Shuler, M.L., and Kargi, F. 2002. Bioprocess Engineering Basic Concepts. In 2nd edition. Prentice Hall PTR, Upper Saddle River, NJ.</p>
					<p class="references">Arán, D., Antelo, J., Lodeiro, P., Macías, F., and Fiol, S. 2017. Use of Waste-Derived Biochar to Remove Copper from Aqueous Solution in a Continuous-Flow System. Ind. Eng. Chem. Res. 56(44): 12755–12762. doi:10.1021/acs.iecr.7b03056.</p>
					<p class="references">Al-Saydeh, S.A., El-Naas, M.H., and Zaidi, S.J. 2017. Copper removal from industrial wastewater: A comprehensive review. J. Ind. Eng. Chem. 56: 35–44. The Korean Society of Industrial and Engineering Chemistry. doi:10.1016/j.jiec.2017.07.026.</p>
					<p class="references">Chiu, H.S.S., Tsang, K.L., and Lee, R.M.L. 1984. Treatment of electroplating wastes. Water Sanit. Asia Pacific Proc. 10th WEDC Conf.: 115–119.</p>
					<p class="references">Leaver-Fay, A., Tyka, M., Lewis, S. M., Lange, O. F., Thompson, J., Jacak, R., Kaufman, K., Renfrew, P. D., Smith, C. A., Sheffler, W., Davis, I. W., Cooper, S., Treuille, A., Mandell, D. J., Richter, F., Ban, Y. E. A., Fleishman, S. J., Corn, J. E., Kim, D. E., … Bradley, P. (2011). Rosetta3: An object-oriented software suite for the simulation and design of macromolecules. Methods in Enzymology, 487(C), 545–574. https://doi.org/10.1016/B978-0-12-381270-4.00019-6</p>
					<p class="references">Kaufmann, K. W., Lemmon, G. H., Deluca, S. L., Sheehan, J. H., & Meiler, J. (2010). Practically useful: What the Rosetta protein modeling suite can do for you. Biochemistry, 49(14), 2987–2998. https://doi.org/10.1021/bi902153g</p>
					<p class="references">How to prepare structures for use in Rosetta. (2020). Retrieved 14 October 2020, from https://www.rosettacommons.org/docs/latest/rosetta_basics/preparation/preparing-structures</p>
					<p class="references">Applications for Macromolecule Design. (2020). Retrieved 14 October 2020, from https://www.rosettacommons.org/docs/latest/application_documentation/design/design-applications</p>
					<p class="references">Relax application. (2020). Retrieved 14 October 2020, from https://www.rosettacommons.org/docs/latest/application_documentation/structure_prediction/relax</p>
					<p class="references">Scoring Tutorial. (2020). Retrieved 14 October 2020, from https://www.rosettacommons.org/demos/latest/tutorials/scoring/scoring</p>
					<p class="references">iGEM Technion 2016, & iGEM TU Eindhoven 2016. (2016). Retrieved 14 October 2020, from http://2016.igem.org/wiki/images/5/59/Rosetta_Guide_for_the_iGEM_Beginner.pdf</p>
					<p class="references">Lawton, T. J., Kenney, G. E., Hurley, J. D., & Rosenzweig, A. C. (2016). The CopC Family: Structural and Bioinformatic Insights into a Diverse Group of Periplasmic Copper Binding Proteins. Biochemistry, 55(15), 2278-2290. doi:10.1021/acs.biochem.6b00175</p>
					<p class="references">van Dijken, K., Prince, Y., Wolters, T., Frey, M., Mussati, G., Kalff, P., Hansen, O., Kerndrup, S., Søndergârd, B., Rodrigues, E.L., and Meredith, S. 1999. Electroplating industry. : 97–128. doi:10.1007/978-94-007-0854-9_9.</p>
					<p class="references">TSMC Corporate Social Responsibility Report 2019. 2019. Retrieved from https://csr.tsmc.com/download/csr/2019-csr-report/english/pdf/e-all.pdf</p>
					<p class="references">Du, X., Li, H., Wang, X., Liu, Q., Ni, J., & Sun, H. (2013). Kinetics and thermodynamics of metal binding to the N-terminus of a human copper transporter, hCTR1. Chemical Communications, 49(80), 9134. doi:10.1039/c3cc45360j</p>
					<p class="references">McLean, B. W., Bray, M. R., Boraston, A. B., Gilkes, N. R., Haynes, C. A., & Kilburn, D. G. (2000). Analysis of binding of the family 2a carbohydrate-binding module from Celludomonas fimi xylanase 10a to cellulose: Specificity and identification of functionally important amino acid residues. Protein Engineering, 13(11), 801–809. https://doi.org/10.1093/protein/13.11.801</p>
					<p class="references">Young, T. R., Wijekoon, C. J. K., Spyrou, B., Donnelly, P. S., Wedd, A. G., & Xiao, Z. (2015). A set of robust fluorescent peptide probes for quantification of Cu<sup>2+</sup> binding affinities in the micromolar to femtomolar range. Metallomics, 7(3), 567–578. doi:10.1039/c4mt00301b</p>
					<p class="references">Sen, P., Bhattacharjee, C., & Bhattacharya, P. (2016). Experimental studies and two-dimensional modelling of a packed bed bioreactor used for production of galacto-oligosaccharides (GOS) from milk whey. Bioprocess and Biosystems Engineering, 39(3), 361–380. https://doi.org/10.1007/s00449-015-1516-2</p>
					<p class="references">Berendsen, et al. 1995. GROMACS: A message-passing parallel molecular dynamics implementation. Comp. Phys. Comm. 91: 43-56.. doi:10.1016/0010-4655(95)00042-E</p>
					<p class="references">National Center for Biotechnology Information (2020). PubChem Compound Summary for CID 16211032, Cellulose. Retrieved October 22, 2020 from https://pubchem.ncbi.nlm.nih.gov/compound/Cellulose.</p>
					<p class="references">Ergun, S., & Orning, A. A. (1949). Fluid Flow through Randomly Packed Columns and Fluidized Beds. Industrial & Engineering Chemistry, 41(6), 1179-1184. doi:10.1021/ie50474a011</p>
					<p class="references">Wu, J., Yu, B., & Yun, M. (2007). A resistance model for flow through porous media. Transport in Porous Media, 71(3), 331-343. doi:10.1007/s11242-007-9129-0</p>
				</div>
			</div> 
				
		</article>
		<accessibility></accessiblity>
    </main>
  <footer></footer>
</body>
</html>


